cylc/flow/cfgspec/globalcfg.py

```diff
@@ -223,8 +223,8 @@
                 DurationFloat(300),
                 desc='''
                     Default for
-                    :cylc:conf:
-                    `flow.cylc[scheduler][mail]task event batch interval`.
+                    :cylc:conf:`flow.cylc
```

Comment:
can handle newlines within the reference string thinggy (the bit in the backqoutes) but cant handle a newline at the beginning of the reference string thinggy.